### PR TITLE
Contribution

### PR DESCRIPTION
**Pull request by mgjean**
> Add two more files;
> 1. HowTo_HideUnhideLinksLevelsGrids.py
> 2. HowTo_ReloadAllLinkDocuments.py

### CODE REVIEW

File: 0-python-code/HowTo_HideUnhideLinksLevelsGrids.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+"""
+Hide Unhide Links Levels Grids
```
Comment (gtalarico) [inline 0-python-code/HowTo_HideUnhideLinksLevelsGrids.py]: I don't fully understand what this does. 
Can you add a long description?

File: 0-python-code/HowTo_HideUnhideLinksLevelsGrids.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+from System.Collections.Generic import List
+from Autodesk.Revit.DB import Transaction
+from Autodesk.Revit.DB import *
+
+doc = __revit__.ActiveUIDocument.Document
+
+
+active_view = doc.ActiveView
+# filter name
+ifilter = "LinkLevelGrid_Quasar"
```
Comment (gtalarico) [inline 0-python-code/HowTo_HideUnhideLinksLevelsGrids.py]: What's Quasar?

File: 0-python-code/HowTo_ReloadAllLinkDocuments.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+"""
+Reload All Link Documents
```
Comment (gtalarico) [inline 0-python-code/HowTo_ReloadAllLinkDocuments.py]: looks great

**Comment by mgjean** [comment]
I do _apologize_ for making you _confuse_. I should annotate clearly. 
Actually, this script can " _Hide And Unhide Levels and Grids that come from Link Documents_ ". Please see the attached _gif_ below. And " _LinkLevelGrid_Quasar_ " is just a **name for filter element**. ;)

Alt Text

File: 0-python-code/HowTo_HideUnhideLinksLevelsGrids.py (hunk 1 above)
Comment (mgjean) [reply 0-python-code/HowTo_HideUnhideLinksLevelsGrids.py]: Ok, i will improve my description.

**Comment by mgjean** [comment]
I closed this PR again.

**Comment by gtalarico** [comment]
Did you give up on the PR? You can just push the changes over and update it :)

**Pull request closed without merging**